packages/jest-matchers/src/index.js

```diff
@@ -35,7 +36,7 @@ if (!global[GLOBAL_STATE]) {
   Object.defineProperty(
     global,
     GLOBAL_STATE,
-    {value: {matchers: Object.create(null), state: {suppressedErrors: []}}},
+    {value: {matchers: Object.create(null), state: {expectedCounts: {}, passedExpectations: [], suppressedErrors: []}}},
```

Comment:
Mentioned that in the PR explanation. Will flip it over to non-object.
I was going to add `assertionsPassing`, and `assertionsFailing` before I realized that didn't make sense.